int-set/src/lib.rs

```diff
@@ -110,22 +163,12 @@ impl<T: Into<u32> + Copy> IntSet<T> {
 impl<T> IntSet<T> {
     /// Create a new empty set (inclusive).
     pub fn empty() -> IntSet<T> {
-        IntSet(Membership::Inclusive(BitSet::empty()))
+        IntSet(Membership::Inclusive(BitSet::empty()), PhantomData::<T>)
     }
 
     /// Create a new set which contains all integers (exclusive).
     pub fn all() -> IntSet<T> {
-        IntSet(Membership::Exclusive(BitSet::empty()))
-    }
-
-    /// Returns an iterator over all members of the set.
-    /// Note: iteration of inverted sets can be extremely slow due to the very large number of members in the set
-    /// care should be taken when using .iter() in combination with an inverted set.
-    pub fn iter(&self) -> impl Iterator<Item = u32> + '_ {
-        match &self.0 {
-            Membership::Inclusive(s) => int_set_iter(s.iter(), false),
-            Membership::Exclusive(s) => int_set_iter(s.iter(), true),
-        }
+        IntSet(Membership::Exclusive(BitSet::empty()), PhantomData::<T>)
```

Comment:
Just out of curiousity, is this constructor used much in harfbuzz?

Reply:
Yes, mostly in parsing the command line flags/public api. It's used when asking for something like --unicodes="*"